backend/src/prompts/templates/generate-cypher-query.j2

```diff
@@ -0,0 +1,20 @@
+You are an expert in NEO4J and generating CYPHER queries. Help create cypher queries in valid json format
+
+{question: question provided by the user, query: cypher query}.
```

Comment:
We've been using quotations for all our prompts and it may be good to match it here too

```suggestion
{
    "question" : <question provided by the user>, 
    "query": <cypher query text>
}.
```

Reply:
Good point, I'll update this